001-IntroToKubernetes/Student/Challenge-01A.md

```diff
@@ -0,0 +1,98 @@
+# Challenge 01 - Path A: Got Containers?
+
+[< Previous Challenge](./Challenge-01.md) - **[Home](../README.md)** - [Next Challenge >](./Challenge-02A.md)
+
+## Introduction
+
+The first step in our journey will be to take our application and package it as a container image using Docker.
+
+## Description
+
+To help ensure everyone can start this hack on the same page and avoid the complexities of installing and setting up Docker, we have provided you with an ARM Template that will deploy a Linux VM into Azure. Once deployed, the Linux VM will have the FabMedical application's source code and all of the Docker & Node JS tools pre-installed.
```

Comment:
The correct spelling of Node JS is Node.js. Might want to change throughout this section

Reply:
Done. (This probably appears in more locations than this.  Really old content!)